unfurler/src/config.ts

```diff
@@ -1,4 +1,16 @@
-const configFile = require('../config.json');
+const fs = require('fs');
+const path = require('path');
+
+const configPath = process.env.UNFURLER_CONFIG || '../config.json';
```

Comment:
can we make the default path `'./config.json'`?

(`require('../config.json')` is relative to this file, but `path.resolve('../config.json')` is relative to the working directory, i.e. one level higher)

Reply:
@mononaut good catch. I think I changed this when it was running off `dist`